### PR TITLE
two dockers

### PR DESCRIPTION
**Pull request by lenisha**
Changes proposed in this pull request:
Add docker files for web and web/python

### CODE REVIEW

File: docker/web-and-data/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# add importer scripts to PATH for easy running in containers
+RUN find /cbioportal/core/src/main/scripts/ -type f -executable \! -name '*.pl'  -print0 | xargs -0 -- ln -st /usr/local/bin
+
+# put config files in this folder if you want to override config
+ENV PORTAL_HOME=/cbioportal
+
+# TODO: fix the workdir-dependent references to '../scripts/env.pl' and do this:
+# RUN find $PWD/core/src/main/scripts/ -type f -executable \! \( -name env.pl -o -name envSimple.pl \)  -print0 | xargs -0 -- ln -st /usr/local/bin
+
```
Comment (khzhu) [inline docker/web-and-data/Dockerfile:46]: @lenisha , got this error with the image

> docker logs cbioportal
Error: Unable to access jarfile webapp-runner.jar

Comment (khzhu) [reply docker/web-and-data/Dockerfile:46]: put back lines to copy war files 
> 
COPY --from=build /cbioportal/portal/target/cbioportal*.war /app.war
COPY --from=build /cbioportal/portal/target/dependency/webapp-runner.jar /webapp-runner.jar

Comment (khzhu) [reply docker/web-and-data/Dockerfile:46]: still not working, have you tested web-and-data docker file, @lenisha? thanks!

Comment (lenisha) [reply docker/web-and-data/Dockerfile:46]: do we need app.war in the loader images?

File: pom.xml [hunk 2]
```diff
@@ -391,7 +391,7 @@
     <dependency>
       <groupId>mysql</groupId>
       <artifactId>mysql-connector-java</artifactId>
-      <version>5.1.16</version>
+      <version>5.1.47</version>
     </dependency>
     <!-- slf4j -->
     <dependency>
```
Comment (khzhu) [inline pom.xml]: great, testing now.

File: docker/web-and-data/Dockerfile (hunk 1 above)
Comment (khzhu) [reply docker/web-and-data/Dockerfile:46]: yes, and web-runner.jar too.

**Comment by lenisha** [comment]
To make it run with Azure MySQL:
- use latest mysql 5.1.x driver
- add db.use_ssl =true flag
```
docker run -d -v /path-to/portal.properties:/cbioportal/portal.properties -p 8086:8080 \
  -e JAVA_OPTS='-Xms2g -Xmx4g -Ddbconnector=dbcp -Ddat.method=none -Dauthorization=false -Dauthenticate=false -Ddb.use_ssl=true -Ddb.user=user@server -Ddb.password= -Ddb.host=server.mysql.database.azure.com -Ddb.portal_db_name=cbio_test -Dsession.service.url=http://localhost/api/sessions/my_portal/' \
  --name cbio cbioportal-web /bin/sh -c 'java ${JAVA_OPTS} -jar webapp-runner.jar /app.war'
 ```

**Comment by khzhu** [comment]
actually I found db.use_ssl in portal.properties file does not matter. Important is 

> -Ddb.use_ssl=true

that has to be set to True if mysql use SSL.

**Pull request closed without merging**